### DISCUSSION TITLE
Time display in web interface

### DISCUSSION THREAD

**Discussion by sk011** [netbox-community/netbox#13023] [Help Wanted!]
### NetBox version

v3.5.3

### Python version

3.8

### Steps to Reproduce

1. Click create item example VLAN
2. Click  on ID Change log 
3. Time dispaly error
system date/time Fri 23 Jun 2023 09:55:09 MSK
date/time in netbox/configuration.py TIME_ZONE = 'UTC'
date/time in DB 
```
postgres=# select NOW();
              now              
-------------------------------
 2023-06-23 09:55:46.821546+03
(1 row)

postgres=# show timezone;
   TimeZone    
---------------
 Europe/Moscow
(1 row)
netbox=# select created,last_updated,vid from ipam_vlan where vid = '1024';
            created            |         last_updated          | vid  
-------------------------------+-------------------------------+------
 2023-06-23 09:55:22.120319+03 | 2023-06-23 09:55:22.120346+03 | 1024


```
![image](https://github.com/netbox-community/netbox/assets/1744917/99f6460e-f188-4d49-a753-3806ad74e62e)

### Expected Behavior

correct time display

### Observed Behavior

time in changelog incorrect regarding OS timezone

**Comment by abhi1693** [comment]
Could you expand more on what you are expecting to be fixed here? I'm not able to understand what fix you want implemented.

**Comment by sk011** [comment]
> Could you expand more on what you are expecting to be fixed here? I'm not able to understand what fix you want implemented.

on screenshot we have section "Change"  filed Time +6 hours regarding UTC timezone or +3 regarding system timezone MSK (that is, in the future)
in the section "Post-Сhange Data" fields created & last_updated look right

**Comment by kkthxbye-code** [comment]
I cannot replicate this. I tried setting postgresql to 'Europe/Moscow' and TIME_ZONE to 'UTC' and everything was displayed correctly. You will need to provide proper replication steps if you want anyone to look at this, however in all likelihood it's a configuration error outside netbox.

**Comment by martinum4** [comment]
>date/time in netbox/configuration.py TIME_ZONE = 'UTC'

maybe you should select the right timezone there too and see if the error persists

**Comment by sk011** [comment]
> > date/time in netbox/configuration.py TIME_ZONE = 'UTC'
> 
> maybe you should select the right timezone there too and see if the error persists

UTC is recomennded timezone and in one section i have correct time in second section error

**Comment by sk011** [comment]
maybe error after change system OS timezone
1  install clear netbox on UTC timezone
2 change system timezone 

how clear cache & reinstall all requirements ?

**Comment by DanSheps** [comment]
@sk011 

Could you please provide clear steps for us to reproduce this, including any changing of:

* System Timezone
* Netbox recommended timezone
* PostgreSQL relevant settings
* User preferences

**Comment by sk011** [comment]
> @sk011
> 
> Could you please provide clear steps for us to reproduce this, including any changing of:
> 
> * System Timezone
> * Netbox recommended timezone
> * PostgreSQL relevant settings
> * User preferences
Now i have 
System Timezone:
```
timedatectl 
               Local time: Вт 2023-06-27 16:29:46 MSK
           Universal time: Вт 2023-06-27 13:29:46 UTC
                 RTC time: n/a                       
                Time zone: Etc/UTC (MSK, +0300)  
```
Netbox conf file UTC
PostgreSQL Europe/Moscow
User preferences - what do you mean? 
![image](https://github.com/netbox-community/netbox/assets/1744917/c35f2233-142b-46fb-b332-a0b8e02578ee)
